benefits/core/migrations/local_fixtures.json

```diff
@@ -62,15 +86,16 @@
       "eligibility_start_template": "eligibility/start--senior.html",
       "claims_scope": "verify:senior",
       "claims_claim": "senior",
-      "supported_enrollment_methods": ["digital", "in_person"]
+      "supported_enrollment_methods": ["digital", "in_person"],
+      "transit_agency_id": 1
```

Comment:
This does work, but to be more consistent with how other ForeignKeys are defined in these fixtures, let's have this as
```
"transit_agency": 1
```
instead.

Same for the rest of the instances of `transit_agency_id` in this file.